nilearn/datasets/struct.py

```diff
@@ -231,6 +231,68 @@ def fetch_icbm152_brain_gm_mask(data_dir=None, threshold=0.2, resume=True,
     return gm_mask_img
 
 
+def fetch_icbm152_brain_wm_mask(data_dir=None, threshold=0.2, resume=True,
```

Comment:
This function has (almost) exactly the same code as `fetch_icbm152_brain_gm_mask`. I would prefer something more DRY like having a private function (`_fetch_icbm152_brain_mask` for example) which would have a `mask_type` parameter (either `wm` or `gm`). Then, the two public functions `fetch_icbm152_brain_gm_mask` and `fetch_icbm152_brain_wm_mask` would call `_fetch_icbm152_brain_mask` with `mask_type` set accordingly.
WDYT?